app/models/order.rb

```diff
@@ -36,11 +36,9 @@ class Order < ActiveRecord::Base
   end
 
   def total_order_price(seller_id=nil)
-    # TODO: come back and talk about the method names
-    # but fwiw Order.price makes sense to me. more detailed explanation in Orderitem model. -J
     items = seller_id ? order_items.select{ |item| item.seller.id == seller_id } : order_items
     array_of_totals = items.map { |item| item.total_item_price }
-    total = array_of_totals.reduce(0) { |sum, current_total| sum += current_total }
+    total = array_of_totals.sum
```

Comment:
Do you want to combine 40 and 41 so it just says:
 
items.map { |item| item.total_item_price }.sum

If not, that's okay, but I'd be inclined to make it just one line.